src/components/ProgressModalPage/index.module.scss

```diff
@@ -14,7 +14,7 @@
     width: 100%;
     height: 100%;
     display: flex;
-    z-index: 520;
+    z-index: 509;
```

Comment:
modal's z-index is 510, so we need to decrease this

Reply:
I think maybe we can make it smaller, leave more modification space, wdyt?

Reply:
nope. page header's z-index is 509. this is what happens when you set this smaller 
![image](https://user-images.githubusercontent.com/93185957/202334058-dc6f2b94-4ad1-4857-b09f-cb80827ba8fa.png)